csvapi/type_tester.py

```diff
@@ -25,6 +25,8 @@ def __init__(self, **kwargs):
         super(Time, self).__init__(**kwargs)
 
     def cast(self, d):
+        if d is None:
+            return d
```

Comment:
Is this needed? The next line casts to a string before trying to match the regex.

Reply:
tests fail w/o this so I guess so :-)